### DISCUSSION TITLE
Image tags policy

### DISCUSSION THREAD

**Discussion by mqmr** [google/mtail#883] [Q&A] [answered]
[NOT A CONTRIBUTION]

Hello,

Could you please clarify the image tag policy which is used for releasing mtail images?
The question comes from an observation that previous tags and layers disappear from the repo with a release of a new mtail version. That causes some issues with mtail deployments. Using the "latest" tag is out of option in our case. 

Thank you

**Comment by jaqx0r** [comment, marked as answer]
The workflow .github/workflows/remove-old-package-versions.yml keeps the last 10 versions around.

If you'd like a different policy please propose a PR to change it!

**Comment by mqmr** [comment]
Thanks for the reply!

Briefly looking into "actions/delete-package-versions" action docs, it seems there are two possible  options:
* https://github.com/actions/delete-package-versions?tab=readme-ov-file#delete-all-pre-release-versions-except-y-latest-pre-release-package-versions
* https://github.com/actions/delete-package-versions?tab=readme-ov-file#delete-all-except-y-latest-versions-while-ignoring-particular-package-versions
* https://github.com/actions/delete-package-versions?tab=readme-ov-file#delete-oldest-x-number-of-versions-while-ignoring-particular-package-versions

Preferably keeping release versions and removing RCs would be an option.
